foreman/client.py

```diff
@@ -21,6 +21,14 @@
 except ImportError:
     PLUGINS = []
 
+try:
+    from requests.packages.urllib3.util.retry import Retry
+    RETRIES = 5
+    DEFAULT_SLEEP = 0.5
+    with_retry = True
+except ImportError:
+    with_retry = False
+
```

Comment:
Is it required to use the Retry from urllib3? Isn't it enough to use the max_retries parameter of the HTTPAdapter and skip all this import logic?

http://docs.python-requests.org/en/latest/api/#requests.adapters.HTTPAdapter